packages/expo-background-fetch/android/build.gradle

```diff
@@ -0,0 +1,73 @@
+
+buildscript {
+    repositories {
+        jcenter()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.3'
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'maven'
+
+group = 'host.exp.exponent'
+version = '1.0.2'
+
+// Upload android library to maven with javadoc and android sources
+configurations {
+    deployerJars
+}
+
+// Creating sources with comments
+task androidSourcesJar(type: Jar) {
+    classifier = 'sources'
+    from android.sourceSets.main.java.srcDirs
+}
+
+// Put the androidSources and javadoc to the artifacts
+artifacts {
+    archives androidSourcesJar
+}
+
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            configuration = configurations.deployerJars
+            repository(url: mavenLocal().url)
+        }
+    }
+}
+
+android {
+    compileSdkVersion 26
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 26
+        versionCode 4
+        versionName "1.0.2"
```

Comment:
Not intended - I guess it's just because the script for generating new modules is not downgrading versions to `1.0.0` - see https://github.com/expo/universe/issues/3006
So it generates new modules with same version os `expo-module-template` package 😉 Thanks for catching that up!